package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "fullstack_shop",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "node src/index.js",
+    "dev": "nodemon src/index.ts",
```

Comment:
"dev": "nodemon -L -e js,ts,json --exec \"npm run start\"",
 "start": "node -r dotenv/config --enable-source-maps dist/app.js dotenv_config_path=nazwa_pliku_z_envami",
The -r flag is required to preload environment variables